src/app/components/card/CardButton.vue

```diff
@@ -20,10 +20,16 @@
           </span>
         </div>
         <div class="opacity-60">
-          <slot v-if="$slots.iconSecondary" name="iconSecondary" />
+          <slot v-if="$slots.iconS" name="iconS" />
           <div v-else-if="props.to">
-            <IHeroiconsArrowTopRightOnSquare v-if="isExternal" />
-            <IHeroiconsChevronRight v-else />
+            <img
```

Comment:
[`<NuxtImg>`](https://image.nuxt.com/) provides us with several optimizations and should be used in all places, except where we explicitly don't want it. But in this case I think we can use slots and imported SVGs sources from the `src/app/assets` directory.

Reply:
@huzaifaedhi22 the first sentence was meant as a general thought to have when using `img`. My second sentence is more important here, as we're working with SVGs, not raster graphics (which is where `NuxtImg` excels at. For vector graphics, something like `svgo` (or [`nuxt-svgo`](https://nuxt.com/modules/nuxt-svgo)) may be something we can try out to have several optimizations readily included.